src/node/rpc/handlerregistry.h

```diff
@@ -87,23 +88,48 @@ namespace ccf
      * @param rw Flag if method will Read, Write, MayWrite
      * @param params_schema JSON schema for params object in requests
      * @param result_schema JSON schema for result object in responses
+     * @param require_client_signature If true, client request must be signed
+     * @param execute_locally If true, request is executed without consensus
+     * (PBFT only)
      */
     void install(
       const std::string& method,
       HandleFunction f,
       ReadWrite rw,
       const nlohmann::json& params_schema = nlohmann::json::object(),
       const nlohmann::json& result_schema = nlohmann::json::object(),
+      bool require_client_signature = false,
       bool execute_locally = false)
```

Comment:
I agree with @olgavrou - these should both get named enums rather than bools.

Related, I think we should rewrite how we set these fields in a future PR: https://github.com/microsoft/CCF/issues/914

Reply:
Agreed. The CLI11 way would be the neatest. Are you happy for me to leave it as is in this PR and address the whole thing in #914?

Reply:
I think we can assume we're going to do #914 and then, rather than unnecessarily complicated named enums, we can just add comments in `commonhandlerregistry` or the few other places we actually set these bools.